recipes/libdispatch/all/conanfile.py

```diff
@@ -60,4 +57,6 @@ def package(self):
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
         if self.settings.os == "Linux":
+            self.cpp_info.libs.sort()
```

Comment:
I see two libs: "libBlocksRuntime.so, libdispatch.so". Better to be explicit here and to set `self.cpp_info.libs` to the correctly ordered list.

Reply:
Agree